scripts/release/Release.ts

```diff
@@ -21,11 +21,12 @@ async function main(): Promise<void> {
     spawnSync("npm", ["run", "release"], { cwd: projectPackage.path });
     await createChangeLog(description, projectPackage.version, projectPackage.path);
 
-    zipTestProjects(projectPackage.path);
-    releaseWithAuth(await getAuth(), projectPackage);
+    await zipTestProjects(projectPackage.path);
```

Comment:
maybe `Promise.all` to paralellize the promises

Reply:
Order is required zip before release, maybe generating changelog in parallel... though that is not make it faster...